iceoryx_posh/include/iceoryx_posh/internal/runtime/posh_runtime_impl.hpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
This is a copy of posh_runtime.hpp. Unfortunately github detects it as new file. The history is preserved and the file should be viewed with an external diff tool